React_pakadmin/tar11_5/src/PickAColors.js

```diff
@@ -0,0 +1,45 @@
+import React, { useState } from "react";
+import tinycolor from "tinycolor2";
+
+export default function PickAColors() {
+  const [divColor, setDivColor] = useState();
+  const triggers = [40, 30, 20, 10, 0, -10, -20, -30, -40];
```

Comment:
You don't need to use a special color library, in CSS you can define color in the HSL format:
Color, Hue, Saturation: https://css-tricks.com/hsl-hsla-is-great-for-programmatic-color-control/
As a rule of thumb, it is always better to write a few extra lines yourself than to add a package.